spec/lib/fields/belongs_to_spec.rb

```diff
@@ -66,4 +66,15 @@
       end
     end
   end
+
+  describe "foreign_key option" do
+    it "determines what foreign key is used on the relationship for the form" do
+      association =
+        Administrate::Field::BelongsTo.with_options(
+          foreign_key: "customers_uuid",
+        )
+      field = association.new(:customers, [], :show)
+      expect(field.permitted_attribute).to eq("customers_uuid")
```

Comment:
Could we break up the phases of this test with newlines? (https://robots.thoughtbot.com/four-phase-test)